UserInterface/Dialogues/PreferencesDialog.cpp

```diff
@@ -29,8 +29,9 @@
 
 #include <QTranslator>
 
-OpenInfraPlatform::UserInterface::PreferencesDialog::PreferencesDialog(OpenInfraPlatform::UserInterface::View* view, QWidget *parent /*= nullptr*/) :
-	ui_(new Ui::PreferencesDialog),
+OpenInfraPlatform::UserInterface::PreferencesDialog::PreferencesDialog(OpenInfraPlatform::UserInterface::View* view, QWidget* parent /*= nullptr*/) :
+
+	ui_(new PreferencesDialog(view, parent)),

```

Comment:
What is the reason for these changes?

Reply:
Thank you for a hint. I have tried to find a reason, why OpenInfraPlatform.UI is not working. Now I know why 👍

Reply:
Just revert these changes, as they should not have been there :D